src/utils/fetchCuries.js

```diff
@@ -2,6 +2,7 @@ import API from '~/API';
 
 export default async function fetchCuries(entity, displayAlert, cancel) {
   // Get list of curies that match this search term
+  console.log('lookup ${entity}');
```

Comment:
If you want to include the actual entity variable in this log message, I believe you need to use backticks here instead of single quotes.

Reply:
Yeah, didn't mean to commit those console messages anyway.